content/technotes/2020-03-19-parzer/index.Rmd

```diff
@@ -0,0 +1,157 @@
+---
+slug: "parzer"
+title: "parzer: parse messy geographic coordinates"
+date: 2020-03-19
+author:
+  - Scott Chamberlain
+description: xxxx.
```

Comment:
delete `description:` or leave value blank otherwise "xxx." will show up in twitter card of tweets that share the link

Reply:
![Screen Shot 2020-03-19 at 12 40 33 PM](https://user-images.githubusercontent.com/11927811/77108129-eb539100-69de-11ea-803f-fde3703c5dce.png)

Reply:
filled in the description